modules/cnv-creating-storage-class.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-configuring-local-storage-for-vms.adoc
+
+[id="cnv-creating-storage-class_{context}"]
+= Creating a StorageClass object
+
+When you create a `StorageClass` object, you set parameters that affect the
+dynamic provisioning of PersistentVolumes (PV) that belong to that storage class. +
+[NOTE]
+====
+You cannot update a `StorageClass` object's parameters after you create it.
+====
+
+.Procedure
+
+. Create a YAML file for defining the storage class. For example:
++
+----
+$ touch storageclass.yaml
+----
+
+. Edit the file. For example:
++
+[source,yaml]
+----
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: storage-class-name
+provisioner: <path/to/provisioner>
+reclaimPolicy: Delete <1>
+volumeBindingMode: WaitForFirstConsumer
+----
+<1> The two possible `reclaimPolicy` values are `Delete` and `Retain`. If you
```

Comment:
"reclaimPolicy" is noted here, "name" and "volumeBindingMode" are noted on the "Example: StorageClass YAML for the hostpath provisioner". Why don't we gather these tips in the same place?

Reply:
Great point :) I am updating the PR now.

Reply:
@qwang1 Here's the preview of the updated section: http://file.bos.redhat.com/pousley/010220/hostpath-cdi/cnv/cnv_users_guide/cnv-configuring-local-storage-for-vms.html#cnv-example-storage-class-yaml_cnv-configuring-local-storage-for-vms

I'm glad you caught that!